distribution/std-lib/Base/src/Data/Text/Extensions.enso

```diff
@@ -215,6 +215,10 @@ Text.ends_with suffix = Text_Utils.ends_with this suffix
 Text.contains : Text -> Boolean
 Text.contains sequence = Text_Utils.contains this sequence
 
+## Replaces all occurrences of `oldSequence` within `this` with `newSequence`.
```

Comment:
Please document the semantics more exactly. Like, what's the result of running `'aaa'.replace 'aa' 'b'`?